src/SDKs/SecurityCenter/Management.SecurityCenter/generate.ps1

```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "security/resource-manager" -AutoRestVersion "latest"
```

Comment:
Please confirm that the Sdk was generated using this script.
There must also be a SdkInfo_*.cs file and a .txt log file generated, please commit these files

Reply:
the generated files were generated using this script.
I also added the SdkInfo file and the text file